### PR TITLE
vim-patch:8.2.3914

### PR DESCRIPTION
**Pull request by dundargoc**
#### vim-patch:8.2.3914: various spelling mistakes in comments

Problem:    Various spelling mistakes in comments.
Solution:   Fix the mistakes. (Dominique Pellé, closes vim/vim#9416)
https://github.com/vim/vim/commit/af4a61a85d6e8cacc35324f266934bc463a21673

### CODE REVIEW

File: src/nvim/ex_docmd.c [hunk 1]
```diff
@@ -7489,9 +7489,8 @@ void do_exedit(exarg_T *eap, win_T *old_curwin)
   if ((eap->cmdidx == CMD_new
        || eap->cmdidx == CMD_tabnew
        || eap->cmdidx == CMD_tabedit
-       || eap->cmdidx == CMD_vnew
-       ) && *eap->arg == NUL) {
-    // ":new" or ":tabnew" without argument: edit an new empty buffer
+       || eap->cmdidx == CMD_vnew) && *eap->arg == NUL) {
```
Comment (clason) [inline src/nvim/ex_docmd.c:7492]: unrelated changes here -- is that to keep lint from complaining?

File: src/nvim/getchar.c [hunk 2]
```diff
@@ -1693,7 +1693,7 @@ typedef enum {
   map_result_fail,    // failed, break loop
   map_result_get,     // get a character from typeahead
   map_result_retry,   // try to map again
-  map_result_nomatch  // no matching mapping, get char
+  map_result_nomatch,  // no matching mapping, get char
```
Comment (clason) [inline src/nvim/getchar.c:1696]: same

**Review by clason — CHANGES_REQUESTED** [review]
Missing
* `cindent.c` (renamed to `indent_c.c`)
* `drawline.c` (moved to `screen.c` line 4297)
* `findfile.c` (moved to `file_search.c`, line 610, 1142)
* `misc1.c` (moved to `plines.c`, line 86)
* `scriptfile.c` (moved to ex_cmds2.c`, line 2165)
* `spellsuggest.c` (moved to `spell.c`, line 222)
* `textformat.c` (moved to `ops.c`, line 4126)

File: src/nvim/ex_docmd.c (hunk 1 above)
Comment (dundargoc) [reply src/nvim/ex_docmd.c:7492]: Yep

**Comment by dundargoc** [comment]
Bah, forgot about those files. Thanks for the reminder.

File: src/nvim/indent_c.c [hunk 3]
```diff
@@ -1647,11 +1647,11 @@ void parse_cino(buf_T *buf)
    * an unclosed parentheses. */
   buf->b_ind_unclosed_whiteok = 0;
 
-  /* Indent a closing parentheses under the line start of the matching
-   * opening parentheses. */
+  /* Indent a closing parenthesis under the line start of the matching
+   * opening parenthesis. */
```
Comment (zeertzjq) [inline src/nvim/indent_c.c:1651]:
```suggestion
  // Indent a closing parenthesis under the line start of the matching
  // opening parenthesis.
```

File: src/nvim/indent_c.c [hunk 4]
```diff
@@ -1635,7 +1635,7 @@ void parse_cino(buf_T *buf)
   buf->b_ind_unclosed2 = sw;
 
   /* Suppress ignoring spaces from the indent of a line starting with an
```
Comment (zeertzjq) [inline src/nvim/indent_c.c:1637]:
```suggestion
  // Suppress ignoring spaces from the indent of a line starting with an
```

File: src/nvim/indent_c.c [hunk 5]
```diff
@@ -1635,7 +1635,7 @@ void parse_cino(buf_T *buf)
...
-   * unclosed parentheses. */
+   * unclosed parenthesis. */
```
Comment (zeertzjq) [inline src/nvim/indent_c.c:1638]:
```suggestion
  // unclosed parenthesis.
```

File: src/nvim/indent_c.c [hunk 6]
```diff
@@ -924,7 +924,7 @@ static int cin_isfuncdecl(char_u **sp, linenr_T first_lnum, linenr_T min_lnum)
   while (*s && *s != ';' && *s != '\'' && *s != '"') {
     if (*s == ')' && cin_nocode(s + 1)) {
       /* ')' at the end: may have found a match
```
Comment (zeertzjq) [inline src/nvim/indent_c.c:926]:
```suggestion
      // ')' at the end: may have found a match
```

File: src/nvim/indent_c.c [hunk 7]
```diff
@@ -924,7 +924,7 @@ static int cin_isfuncdecl(char_u **sp, linenr_T first_lnum, linenr_T min_lnum)
...
-       * Check for he previous line not to end in a backslash:
+       * Check for the previous line not to end in a backslash:
```
Comment (zeertzjq) [inline src/nvim/indent_c.c:927]:
```suggestion
      // Check for the previous line not to end in a backslash:
```

File: src/nvim/indent_c.c [hunk 8]
```diff
@@ -924,7 +924,7 @@ static int cin_isfuncdecl(char_u **sp, linenr_T first_lnum, linenr_T min_lnum)
...
        *       #if defined(x) && \
        *		 defined(y)
        */
```
Comment (zeertzjq) [inline src/nvim/indent_c.c:930]:
```suggestion
      //       #if defined(x) && \
      //           defined(y)
```

**Comment by clason** [comment]
> Bah, forgot about those files. Thanks for the reminder.

Still missing one:
* `indent_c.c` line 927

**Comment by zeertzjq** [comment]
>     * `indent_c.c` line 927

What's missing on that line?

**Comment by zeertzjq** [comment]
This PR first need to be squashed though.

**Comment by clason** [comment]
> > ```
> > * `indent_c.c` line 927
> > ```
> 
> 
> What's missing on that line?

Oops, I missed that change in the comment refactor. Never mind, LGTM! I'll squash and merge.

**Comment by clason** [comment]
@dundargoc This change offended lint:
```
  ../src/nvim/indent_c.c: In function ‘cin_isfuncdecl’:
  ../src/nvim/indent_c.c:928:7: error: multi-line comment [-Werror=comment]
    928 |       //       #if defined(x) && \
        |       ^
  cc1: all warnings being treated as errors
  ```
